ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -10,11 +10,54 @@ import LlamaKit
 import Nimble
 import Quick
 import ReactiveCocoa
+import Foundation
+
+private func startSignalProducer<T: Equatable, E: Equatable>(signalProducer: SignalProducer<T, E>, #expectSentValue: T?, sentError expectSentError: E?, #complete: Bool) {
+	startSignalProducer(signalProducer, expectSentValues: expectSentValue.map { [$0] } ?? [], sentError: expectSentError, complete: complete)
+}
+
+private func startSignalProducer<T: Equatable, E: Equatable>(signalProducer: SignalProducer<T, E>, expectSentValues: [T] = [], sentError expectSentError: E?, #complete: Bool) {
```

Comment:
I implemented it using a matcher (really neat API!). What do you guys think? https://github.com/JaviSoto/ReactiveCocoa/commit/c9dbd222e5bb5b380b1838f7ae897512ca737675